pom.xml

```diff
@@ -121,8 +121,8 @@
     <stream.version>2.9.5</stream.version>
     <i18n-util.version>1.0.4</i18n-util.version>
     <guice.version>4.0</guice.version>
-    <zookeeper.version>3.4.14</zookeeper.version>
-    <curator.version>4.0.0</curator.version>
+    <zookeeper.version>3.5.7</zookeeper.version>
+    <curator.version>4.3.0</curator.version>
```

Comment:
HBase uses 4.2.
While it's probably OK to use a newer version, if there is no particular reason to use 4.3, it's better to stick with the version HBase uses.